docker/swarm.docker-compose.yaml

```diff
@@ -0,0 +1,411 @@
+    ports:
+      - "8090:8090/tcp"
+    networks:
+      datagrok:
+        aliases:
+          - cvm
+          - nginx_cvm
+    logging: *default-logging
+
+  grok_compute:
+    image: datagrok/grok_compute:latest
+    stop_grace_period: 2m
+    deploy:
+      replicas: 1
+      mode: replicated
+      endpoint_mode: dnsrr
+      placement: *cvm-placement
+      update_config:
+        <<: *update-config
+        order: start-first
+      restart_policy:
+        <<: *restart-policy
+      rollback_config:
+        <<: *rollback-config
+        order: start-first
+      resources:
+        reservations:
+          cpus: '0.125'
+          memory: 100M
+    healthcheck:
+      <<: *healthcheck
+      test: ["CMD", "wget", "-SqO", "-", "http://localhost:5005/grok_compute/info"]
+    environment:
+      GROK_COMPUTE_NUM_CORES: 2
+    networks:
+      datagrok:
+        aliases:
+          - grok_compute
+    logging: *default-logging
+
+  jupyter_notebook:
+    image: datagrok/jupyter_notebook:latest
+    stop_grace_period: 2m
+    deploy:
+      replicas: 1
+      mode: replicated
+      endpoint_mode: dnsrr
+      placement: *cvm-placement
+      update_config:
+        <<: *update-config
+        order: start-first
+      restart_policy:
+        <<: *restart-policy
+      rollback_config:
+        <<: *rollback-config
+        order: start-first
+      resources:
+        reservations:
+          cpus: '0.125'
+          memory: 100M
+    healthcheck:
+      <<: *healthcheck
+      test: ["CMD", "wget", "-SqO", "-", "http://localhost:5005/notebook/helper/info"]
+    networks:
+      datagrok:
+        aliases:
+          - jupyter_notebook
+    logging: *default-logging
+
+  jupyter_kernel_gateway:
+    image: datagrok/jupyter_kernel_gateway:latest
+    stop_grace_period: 2m
+    entrypoint: ./entrypoint.sh --main "jupyter kernelgateway --config=.jupyter/jupyter_kernel_gateway_config.py" --helper "gunicorn grok_helper:app --timeout 900 --chdir /home/grok/grok_helper -b 0.0.0.0:5005  --access-logfile - --error-logfile - --workers 4"
+    deploy:
+      replicas: 1
+      mode: replicated
+      endpoint_mode: dnsrr
+      placement: *cvm-placement
+      update_config:
+        <<: *update-config
+        order: start-first
+      restart_policy:
+        <<: *restart-policy
+      rollback_config:
+        <<: *rollback-config
+        order: start-first
+      resources:
+        reservations:
+          cpus: '0.125'
+          memory: 300M
+    healthcheck:
+      <<: *healthcheck
+      test: ["CMD", "curl", "-f", "http://localhost:5005/jupyter/helper/info"]
+    networks:
+      datagrok:
+        aliases:
+          - jupyter_kernel_gateway
+    volumes:
+      - jkg_helper_cache:/home/grok/grok_helper/cache
+      - jkg_helper_envs:/home/grok/grok_helper/environments
+      - jkg_conda_envs:/home/grok/miniconda/envs
+      - jkg_kernels:/home/grok/.local/share/jupyter/kernels/
```

Comment:
We need it for dev cause we restart container every night, it is better to remove it for clients